documents/interaction-states.md

```diff
@@ -0,0 +1,51 @@
+# Interaction state machine
+
+During his work, a user can perform different actions, such as moving the mouse, leaving the seat, or typing on the keyboard. Due to the fact that those actions are immediate, it is possible to instantly transition from any state to any other.
+
+Each of the following states is defined in terms of those 3 parameters:
+
+- **Gaze**: workspace(💻), elsewhere (🌲), none (❌);
+- **Voice**: any (❔), present (✔), none (❌);
+- **Input**: any(❔), mouse & keyboard (✔), none (❌).
+
+## States
+
+### Engagement
+
+| Gaze | Voice | Input |
+| ---- | ----- | ----- |
+| 💻   | ❔    | ✔     |
+
+In this state the user is actively engaged in his work through visual focus on the workspace (that is, the monitor or the table in front of him) and active input from mouse and/or keyboard.
+
+### Idling
+
+| Gaze | Voice | Input |
+| ---- | ----- | ----- |
+| 💻   | ❔    | ❌    |
+
+In this state the user is passively engaged in the work, i.e. he is thinking about the problem or sketching on a paper. Due to the lack of the input, no traceable progress can be recognized.
+
+### Conferencing
+
+| Gaze | Voice | Input |
+| ---- | ----- | ----- |
+| 💻   | ✔     | ❌    |
```

Comment:
This state is intersecting with the **Idling** state, since it has "any" for voice. We should make them distinct.

Reply:
Adjusted.